scripts/minter.ts

```diff
@@ -30,12 +30,32 @@ function clean(obj: any) {
   return obj;
 }
 
+function formatRoyaltyInfo(
+  royaltyPaymentAddress: null | string,
+  royaltyShare: string
```

Comment:
are you sure royaltyShare can't be null? 

from my understanding if you remove it from the config.json it might ?

Reply:
if royalty info commented out `config.royaltyPaymentAddress` is `undefined` and set to null here:

https://github.com/public-awesome/stargaze-tools/blob/yubrew/161-allow-empty-royalty-addr/scripts/minter.ts#L52

Is there a better way to write this?